mealie/services/migrations/utils/migration_helpers.py

```diff
@@ -177,3 +177,21 @@ def parse_iso8601_duration(time: str | None) -> str:
         return_strings.append(f"{value} {value_map[unit_key]}")
 
     return " ".join(return_strings) if return_strings else time
+
+
+def _format_time(minutes: int) -> str:
```

Comment:
Can we rename this to `_format_time`? The leading underscore signals that its private, but since its used in multiple modules it should be public.